doc/src/gotchas.rst

```diff
@@ -270,6 +270,36 @@ trick for getting tab completion in the regular Python console.
     See also `What is the best way to create symbols?
     <https://github.com/sympy/sympy/wiki/Faq>`_ in the FAQ.
 
+.. _calling-functions:
+
+Functions
+---------
+
+A function like ``f(x)`` can be created by defining the Function
+and the variable:
+
+    >>> from sympy import Function
+    >>> f = Function('f')
+    >>> x = Symbol('x')
+    >>> f(x)
+    f(x)
+
+If you assign ``f(x)`` to a Python variable `f` you will lose your
+ability to copy and paste that function or to create a function
+with a different argument: ``Function('f')`` is callable, but
+``Function('f')(x)`` is not:
+
+    >>> f1 = Function('f1')
+    >>> f2 = Function('f2')('x')
+    >>> f1(1)
+    f1(1)
+    >>> f2(1)
```

Comment:
It would also be good to show what happens when you output just `f1` and `f2`.

Reply:
Added the outputs.